internal/service/arcregionswitch/plan_test.go

```diff
@@ -0,0 +1,637 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Needs a `_disappears` test to verify out-of-band removals are captured correctly by the read operation.

https://hashicorp.github.io/terraform-provider-aws/running-and-writing-acceptance-tests/#disappears-acceptance-tests